src/components/ui/CaptionInput.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components/macro'
+
+import CircleIcon from './CircleIcon'
+import IconButton from './IconButton'
+import Input from './Input'
+import ListEntry from './ListEntry'
+
+const StyledCaptionInput = styled(ListEntry)`
+  padding: 8px;
+  height: 33px;
+  display: flex;
+  justify-content: space-between;
+`
+
+const CaptionInput = ({ value, onChange, onDelete, onClick, image }) => (
+  <StyledCaptionInput
+    leftIcons={<CircleIcon>{image}</CircleIcon>}
+    primaryText={
+      <Input placeholder="Add a caption..." onChange={onChange} value={value} />
+    }
+    rightIcons={
+      <IconButton
+        size="44"
+        raised={false}
+        icon={<> &times; </>}
```

Comment:
Try to use an icon from styled-icons here.

Reply:
I'll fix my PR before dev night, but I was wondering which one do you recommend. I couldn't find any within solid box icons and the other one (it's slipping my mind). There's a couple "x" or crosses in other boxicon categories though, but we don't have them in our project.

Reply:
Do you mean in other icon libraries (not boxicons)?

Reply:
Oops nvm